libs/common-const/src/tokens.ts

```diff
@@ -320,19 +422,24 @@ export const COW: Record<SupportedChainId, TokenWithLogo> = {
   [SupportedChainId.MAINNET]: COW_TOKEN_MAINNET,
   [SupportedChainId.GNOSIS_CHAIN]: COW_TOKEN_XDAI,
   [SupportedChainId.ARBITRUM_ONE]: COW_TOKEN_ARBITRUM,
+  [SupportedChainId.BASE]: COW_TOKEN_BASE,
   [SupportedChainId.SEPOLIA]: COW_TOKEN_SEPOLIA,
 }
 
-export const GNO: Record<SupportedChainId, TokenWithLogo> = {
+export const GNO: Record<SupportedChainId, TokenWithLogo | null> = {
   [SupportedChainId.MAINNET]: GNO_MAINNET,
   [SupportedChainId.GNOSIS_CHAIN]: GNO_GNOSIS_CHAIN,
   [SupportedChainId.ARBITRUM_ONE]: GNO_ARBITRUM_ONE,
+  [SupportedChainId.BASE]: null,
   [SupportedChainId.SEPOLIA]: GNO_SEPOLIA,
 }
 
 const SDAI_GNOSIS_CHAIN_ADDRESS = '0xaf204776c7245bf4147c2612bf6e5972ee483701'
 const GBPE_GNOSIS_CHAIN_ADDRESS = '0x5cb9073902f2035222b9749f8fb0c9bfe5527108'
 
+// Not used for fees
```

Comment:
As this is a const file, and you are referring to the business logic. I would not add this comment here